### PR TITLE
Add table element finder in private API

### PR DESCRIPTION
**Pull request by aaltat**

### CODE REVIEW

**Comment by aaltat** [comment]
This is for #911

File: test/unit/api/test_public_api_objects.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from SeleniumLibrary import SeleniumLibrary
+from SeleniumLibrary.locators import ElementFinder
+from SeleniumLibrary.locators import TableElementFinder
```
Comment (pekkaklarck) [inline test/unit/api/test_public_api_objects.py]: Last two lines could (and IMHO should) be combined:

    from SeleniumLibrary.locators import ElementFinder, TableElementFinder

File: src/SeleniumLibrary/__init__.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
                                       TableElementKeywords,
                                       WaitingKeywords)
 from SeleniumLibrary.locators import ElementFinder
+from SeleniumLibrary.locators import TableElementFinder
```
Comment (pekkaklarck) [inline src/SeleniumLibrary/__init__.py]: Could (and IMHO should) be combined with the above import:

    from SeleniumLibrary.locators import ElementFinder, TableElementFinder

**Review by pekkaklarck — APPROVED** [review]
Code changes look good to me. Always nice when the diff has more red than green. I added two nitpick comments about imports, though.

There were lot of changes to tests, apparently due to indentation changes. Didn't bother to look at them too closely now, but tests seem to pass so they ought to be good too.

**Comment by aaltat** [comment]
The changes in the unit test are caused because the ElementFinder is not anymore a instance of the class. Now it's inherited and comes through few properties and mocking was easier this way.

File: test/unit/api/test_public_api_objects.py (hunk 1 above)
Comment (aaltat) [reply test/unit/api/test_public_api_objects.py]: Done

**Comment by aaltat** [comment]
Fixed the imports and I should really stop doing imports that way.

**Pull request merged**